Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
```

Comment:
The tests did not run during the Docker build unless I used --exe. I'm not sure if there is another way around this. I can look into it.